### PR TITLE
updated download limit error text

### PR DESCRIPTION
**Pull request by yxlee61**
https://github.com/WSWCWaterDataExchange/WestDAAT/issues/220

image

### CODE REVIEW

**Comment by yxlee61** [comment]
> @yxlee61 Looks good. Should the "Cancel" button be in the middle and says "Okay" or "Agree" instead of Cancel?

Changed the text to "Okay" when there's a download error. I think it's pretty common to have buttons on aligning right. Would this be preferable?
New:
image
image

Old:
image

**Comment by amabdallah** [comment]
@yxlee61 nice work! so actually let's go with the middle one with Okay on the right (much better than the left). We have a hard limit on 100K records, so users cant download more than that. So they have no option, thus OK or Agree.

**Comment by amabdallah** [comment]
Oh wait, it seems you got it right and I didnt pay attention to Old and New. So all good.

**Comment by yxlee61** [comment]
@tarrball don't merge this yet. there's a timing bug regarding the modal title.

**Review by tarrball — APPROVED** [review]

File: src/DashboardUI/src/components/DownloadModal.tsx [hunk 1]
```diff
@@ -105,17 +111,23 @@ function DownloadModal(props: DownloadModalProps) {
...
       <Modal.Body>
       { !isAuthenticated && <p>
           Sign up <strong>completely free</strong> or login for download access of up to 100,000
           water rights points.
         </p>}
-        {/* display error message */}
-        { isAuthenticated && isError &&
+        {/* display generic error message */}
+        { isAuthenticated && isError && !isDownloadLimitError &&
           <p>An error occurred while attempting to download the data. Please adjust your filters to reduce the data-set and try again.</p>}
```
[the remark this answers is not in the document]
Comment (yxlee61) [reply src/DashboardUI/src/components/DownloadModal.tsx]: the file you commented on is outdated. that code is removed.

**Pull request merged**